CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
 
+## [1.1.5] - 2023-03-29
+### Changed
+* Improve clusterHostInstancePattern example in README.md.
```

Comment:
```suggestion
* Improved `clusterHostInstancePattern` example in README.md.
```

Reply:
More of a general comment regarding the documentation changes, are we also able to link to the affected section here as a hyperlink?

Reply:
Done!

Reply:
Yup! Should be able to. Only thing is that it won't highlight the diffs.

Reply:
Done! The Maven Central URL changes had multiple updates, so I linked each one.